src/navigation/AppNavigator.tsx

```diff
@@ -73,6 +74,10 @@ function DashboardStackNavigator() {
 const Drawer = createDrawerNavigator<DrawerParamList>();
 
 function DrawerNavigator() {
+  useNavigationState((s) => {
```

Comment:
This is not used, did you add it for debugging?

Reply:
oh yes thanks for noticing. will take it out.